platforms/android/tangram/src/main/java/com/mapzen/tangram/MapController.java

```diff
@@ -1380,9 +1422,14 @@ public void onDrawFrame(final GL10 gl) {
         }
 
         boolean viewComplete;
+        boolean isCameraEasing;
         synchronized(this) {
             viewComplete = nativeUpdate(mapPointer, delta);
-            nativeRender(mapPointer);
+            isCameraEasing = nativeRender(mapPointer);
+        }
+
+        if (isCameraEasing) {
+            mapRegionChangeStateManager.setMapRegionState(MapRegionChangeStateManager.MapRegionChangeState.ANIMATING);
```

Comment:
This executes a listener callback on the GL thread, we should run it on the UI thread instead.